starknet_py/serialization/serialization_test.py

```diff
@@ -133,3 +152,27 @@ def test_event_serialization():
     assert TupleDataclass.from_dict(payload) == serializer.deserialize(
         expected_serialized
     )
+
+
+def test_event_serialization_v1():
+    serializer = serializer_for_event(parsed_abi_v1.events["Approval"])
+
+    approval = {"owner": 1, "spender": 2, "value": 3}
+    serialized = serializer.serialize(approval)
+    assert serialized == [1, 2, 3, 0]
```

Comment:
What does `0` at the end mean here?

Reply:
I think it's related to that, the value is `core::integer::u256`, so this 0 is added to save info about the type.  @THenry14 could you confirm?

Reply:
If I remember correctly this is partly true. I think this is because the value is u256, and this cannot fit into single felt - that is why this is being split into `high` and `low` value (and 3 is very small, so its part can fit into felt, leaving the other part as 0).